nameko/web/server.py

```diff
@@ -115,7 +116,8 @@ def get_wsgi_server(
             sock.getsockname(),
             wsgi_app,
             protocol=protocol,
-            debug=debug
+            debug=debug,
+            log=getLogger('wsgi')
```

Comment:
this is a pretty generic name. the most common convention i know is to namespace by module (`getLogger(__name__)`). if you want further customisability you would typically add further namespacing below with another dot